### PR TITLE
(fixes issue #910) Add known mappings to access key logic

### PR DESCRIPTION
**Pull request by flodolo**
Fixes issue #910

### CODE REVIEW

File: app/models/accesskeys.php [hunk 1]
```diff
@@ -45,36 +45,53 @@ function ($entity) {
...
+        $entity = $known_mappings[$ak_string_id];
+        // Check if the label is translated
+        if (isset($target[$entity]) && ! empty($target[$entity])) {
+            $current_ak = $target[$ak_string_id];
+            if (($current_ak == '') || (mb_stripos($target[$entity], $current_ak) === false)) {
+                $ak_results[$ak_string_id] = $entity;
             }
+        }
+    } else {
+        foreach ($ak_labels as $ak_label) {
```
Comment (flodolo) [inline app/models/accesskeys.php]: There is no change in the else clause, it's just the existing code indented.

**Comment by flodolo** [comment]
No clue why but Travis today time outs on PHP 5

**Comment by TheoChevalier** [comment]
Looks good! While we’re there, could you also add this one? (current false positive in French) `'extensions/irc/chrome/pref-irc.dtd:pref-irc.open.accesskey' => 'extensions/irc/chrome/pref-irc.dtd:pref-irc.open.label'`

**Comment by flodolo** [comment]
Added 👍

**Pull request merged**